G/GANAK/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GANAK"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/meelgroup/ganak.git", "36fa4b35c8e3cb588a3a76a87624d97ae7f18285")
+    DirectorySource("bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ganak/
+
+# add all dependencies for linking without 'allow undefined'
+atomic_patch -p1 ../patches/alllibs.patch
+
+# make sure char in LogTable is signed on all platforms (needed for storing -1)
+atomic_patch -p1 ../patches/logtab_signed.patch
```

Comment:
I think these patches should be upstreamed?

Reply:
@benlorenz made PR's with GANAK:
- https://github.com/meelgroup/ganak/pull/23
- https://github.com/meelgroup/ganak/pull/24
- https://github.com/meelgroup/ganak/pull/25